tests/Native/AST.h

```diff
@@ -106,3 +106,13 @@ class Atomics
 # endif
 #endif
 };
+
```

Comment:
I should explain what implicit declarations are. These are declarations which are not written at all but are instead generated by the compiler. The most common examples are the empty constructor, copy constructor, assigment operator and destructor in C++ classes. So in order to test implicity, just select any class without any of these members explicitly defined in code, and assert that IsImplicit is true. For example, let's say you have this class:

class ImplicitCtor
{
};

Then you can do Assert.IsTrue(classes.Find("ImplicitCtor").Constructors.First(c => c.Parameter.Count ==0).IsImplicit);